Makefile

```
@@ -148,26 +148,26 @@ lint-install:
 
 .PHONY: lint
 lint:
-	ifeq (, $(shell which golangci-lint))
-		$(info golangci-lint can't be found, please run: make lint-install)
-		exit 1
-	endif
+	@which golangci-lint >/dev/null 2>&1 || { \
+		echo "golangci-lint not found, please run: make lint-install"; \
+		exit 1; \
+	}
 
 	golangci-lint run
 
-	ifeq (, $(shell which markdownlint))
-		$(info markdownlint can't be found, please run: make lint-install)
-		exit 1
-	endif
+	@which markdownlint >/dev/null 2>&1 || { \
+		echo "markdownlint not found, please run: make lint-install"; \
+		exit 1; \
+	}
 
-	markdownlint-cli
+	markdownlint
```

```suggestion
	markdownlint .
```
The command expects a directory or file, I wonder if running golangci-lint and markdownlint right after each other is too noisy it does fill up the terminal. Maybe worth breaking into separate commands?

---

Well until we fix all the linting issues, markdownlint will never run, right?

---

Good point I didn't realize that, I am ok with keeping it as it is hopefully we can get the go linter issues to zero sooner then later.